clusters/l3-sqnc/capacity/identity-service/values.yaml

```diff
@@ -0,0 +1,24 @@
+logLevel: info
+selfAddress: 5Gh5cFC3sPS7pK8SN41Ytp6VFsMyPpvKxtrrcLpkf19u9W83
+externalSqncNode:
+  host: magenta-sqnc-node-0
+  port: 9944
+authType: EXTERNAL
+externalOrigin: "https://capacity.dsch-l3.com"
+ingress:
+  ingressClassName: nginx-capacity
+  hostname: capacity.dsch-l3.com
+  paths:
+    - path: /(v1/members(/.*)?)
+      pathType: Prefix
+    - path: /(v1/self(/.*)?)
+      pathType: Prefix
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
```

Comment:
I actually think this is redundant as the default values in the chart are `/v1/members` and `/v1/self` and as its a prefix and hosted at the root of the subdomain, we do not need to rewrite.
```suggestion
```